index.js

```diff
@@ -52,8 +52,13 @@ function basepath(config, service, opts) {
 
 function cljsLambdaBuild(serverless, opts) {
   const fns = slsToCljsLambda(serverless.service.functions, opts);
+  const compiler = _.get(serverless.service, 'custom.cljs-compiler');
+  if (process.env.SLS_DEBUG) {
+    serverless.cli.log(`Compiler is set to: ${compiler}`);
+  }
+
```

Comment:
This is gratuitous, it will fail immediately if the compiler is incompatible with the project.